core/cmd/run_xchain_cmd.go

```diff
@@ -8,11 +8,9 @@ import (
 	"syscall"
 
 	"github.com/river-build/river/core/xchain/server"
-
-	"github.com/spf13/cobra"
 )
 
-func run() error {
+func runXChain() error {
```

Comment:
Same as for `runStreamMode` I would like to see this function being moved to the `xchain` package and that the run xchain command directly calls `xchain.Run(cmdConfig)`.

Reply:
same as above